CHANGELOG.md

```diff
@@ -391,11 +391,11 @@ You can now initialize individual components like so:
 
 - Restructure project to enable it to be published as a single package
 
-  The [new project structure]((./src/README.md)) matches our ITCSS inspired layers and is published as a single package as `@govuk-frontend/frontend`.
+  The [new project structure]((./src/README.md)) matches our ITCSS inspired layers and is published as a single package as `@govuk-frontend/frontend` (NOTE: package is called `govuk-frontend` as of release `v0.0.32`).
 
   You will need to:
 
-  - Update your npm dependencies to use `@govuk-frontend/frontend`
+  - Update your npm dependencies to use `@govuk-frontend/frontend` (NOTE: package is called `govuk-frontend` as of release `v0.0.32`)
```

Comment:
i would keep old changelog as it is and at the top of the 32 release add 
- We have changed the name of package. It's now published as` govuk-frontend` on npm (or something similar)